cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -190,6 +192,7 @@ func (s *CMServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.RootCAFile, "root-ca-file", s.RootCAFile, "If set, this root certificate authority will be included in service account's token secret. This must be a valid PEM-encoded CA bundle.")
 	fs.BoolVar(&s.EnableHorizontalPodAutoscaler, "enable-horizontal-pod-autoscaler", s.EnableHorizontalPodAutoscaler, "Enables horizontal pod autoscaler (requires enabling experimental API on apiserver). NOT IMPLEMENTED YET!")
 	fs.BoolVar(&s.EnableDeploymentController, "enable-deployment-controller", s.EnableDeploymentController, "Enables deployment controller (requires enabling experimental API on apiserver). NOT IMPLEMENTED YET!")
+	fs.BoolVar(&s.EnableHostPathProvisioner, "enable-hostpath-provisioner", s.EnableHostPathProvisioner, "Enables HostPath PersistentVolume provisioning for development purposes. This *WILL NOT WORK* in a multi-node cluster.  This is for local development and testing only where master and Kubelet are on a single node.")
```

Comment:
Less is more and I don't mind getting rid of it, so long as we can think of another way to toggle this behavior on in limited ways.

Nil cloud provider is insufficient because the cluster could be running on bare metal.

We need another trigger.  Env var tucked away?